filament/src/details/Renderer.cpp

```diff
@@ -968,11 +968,54 @@ void FRenderer::renderJob(RootArenaScope& rootArenaScope, FView& view) {
     const auto [structure, picking_] = ppm.structure(fg,
             passBuilder, renderFlags, svp.width, svp.height, {
             .scale = aoOptions.resolution,
-            .picking = view.hasPicking()
+            .picking = view.hasPicking() && !view.isTransparentPickable()
     });
-    const auto picking = picking_;
+    auto picking = picking_;
 
     if (view.hasPicking()) {
+        if (view.isTransparentPickable()) {
+            struct PickingRenderPassData {
+                FrameGraphId<FrameGraphTexture> depth;
+                FrameGraphId<FrameGraphTexture> picking;
+            };
+            auto& pickingRenderPass = fg.addPass<PickingRenderPassData>("Picking Render Pass",
+                [&](FrameGraph::Builder& builder, auto& data) {
+                    bool const isES2 = mEngine.getDriverApi().getFeatureLevel() == FeatureLevel::FEATURE_LEVEL_0;
```

Comment:
nit: maybe rename `isES2` to `isFL0` so that we have less backend related details here.

Reply:
Also this line is a little long.  Filament style is <=100 chars per line.

Reply:
Fixed. Thanks for the review!